src/components/CountryDetail.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import { useNavigate } from 'react-router-dom';
+import style from './components.module.css';
+
+function CountryDetail(props) {
```

Comment:
- [ ] Kindly consider following the same design layout, The Country flag and Country Name should be on the same line

Reply:
Hello @Ismailco  you have reviewed the old commit please kindly look at the files changed you will i have updated the layout and also removed the majority of those linters!

Reply:
@Pamphilemkp, yes, you changed the layout but now the title is under the flag, it should be beside it, as the previous reviewer mentioned